lib/logger.js

```diff
@@ -39,6 +39,11 @@ x.test = function (props) {
 		return;
 	}
 
+	if (props.skip) {
+		log.write('  ' + chalk.cyan('- ' + props.title));
```

Comment:
I am afraid `.dim` will make it less noticeable, and the purpose of `.skip()` is to temporarily disable it. So if we keep it bright, it will stay in developer's focus, that something needs fixing.